StarVMC/StarAgmlLib/AgMath.h

```diff
@@ -86,7 +97,11 @@ class AgPower
 Float_t operator*( const Float_t &A, const AgPower<Int_t> &power );
 Float_t operator*( const Float_t &A, const AgPower<Float_t> &power );
 
+}; // AgMath  
+
+
 //Float_t operator^(const Float_t &a, const Float_t &b);
 using namespace std;
+using namespace AgMath;
```

Comment:
Doing this in a header file is of course a very bad practice, especially with the std namespace. I assume this header will never be public